External/FEXCore/Source/Interface/Core/LookupCache.h

```diff
@@ -239,7 +240,11 @@ class LookupCache {
   };
 
 
-  std::map<BlockLinkTag, std::function<void()>> BlockLinks;
+  std::pmr::monotonic_buffer_resource BlockLinks_mbr;
```

Comment:
Should have a small comment here to explain why the map itself is allocated through an allocator.

Reply:
Added a comment.